coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -43,6 +43,10 @@
         'file_arg_template': '+{line},{column} {filename} ',
         'gui': False
     },
+    'emacs': {
+        'file_arg_template': '+{line}:{column} {filename}',
+        'gui': False
+    },
```

Comment:
I'm not an emacs user myself, but I read that some people have emacs running in the background as a server to reduce startup times. In that case they'd open a file with some kind of `emacsclient` command or similar.

If this is so I'd add it here, too.